### ISSUE TITLE
where is the loss function？

### ISSUE THREAD

**Issue by limiao766** [JiarunLiu/Swin-UMamba#9]
Hello, I checked your training code, but I didn’t find where to set the loss function. Could you please explain where it is?

**Comment by JiarunLiu** [comment]
Hi @limiao766, the loss function is initialized in `nnUNetTrainer.py`. We follow the default settings of the loss function in nnU-Net.

https://github.com/JiarunLiu/Swin-UMamba/blob/2c1f72e1975cef6621f239117efdc954237c6952/swin_umamba/nnunetv2/training/nnUNetTrainer/nnUNetTrainer.py#L351-L361
